tracer/src/Datadog.Trace/Configuration/DynamicConfigurationManager.cs

```diff
@@ -66,6 +64,8 @@ private static void OnConfigurationChanged(ConfigurationBuilder settings)
             var headerTags = TracerSettings.InitializeHeaderTags(settings, ConfigurationKeys.HeaderTags, oldSettings.HeaderTagsNormalizationFixEnabled);
             // var serviceNameMappings = TracerSettings.InitializeServiceNameMappings(settings, ConfigurationKeys.ServiceNameMappings);
 
+            var globalTags = settings.WithKeys(ConfigurationKeys.GlobalTags).AsDictionary();
```

Comment:
These are the "fallback" values when we're using dynamic config, but no tags are specified, right? In which case, shouldn't we include the `DD_TRACE_GLOBAL_TAGS` fallback here too?